File: 지민/1주차/code_10870.py
#10870 피보나치 수 5

def fibo(n):
   result = {}
   for x in range(n+1):
      if x ==0:
         result[0] = 0 
      elif x ==1:
         result[1] = 1
      else:
         result[x] = result[x-1] + result[x-2]
      
   return result.get(n) #맨 마지막 값 가지고 오기 


print(fibo(int(input())))

File: 찬영/1주차/code_10870.c
#include<stdio.h>
int fibo(int n) {
	if (n == 0)return 0;
	if (n == 1)return 1;
	return fibo(n - 1) + fibo(n - 2);
}
int main() {
	int result, n;
	scanf("%d", &n);
	result = fibo(n);
	printf("%d\n", result);
}